documentation/src/test/java/org/hibernate/validator/referenceguide/chapter12/constraintapi/ConstraintApiTest.java

```diff
@@ -20,6 +20,7 @@
 import org.hibernate.validator.constraintvalidators.RegexpURLValidator;
 import org.junit.Test;
 
+@SuppressWarnings("deprecation")
```

Comment:
Maybe we should use the new `getter()` and `field()` and check the documentation is up to date?

(same comment for all the deprecation tags added in API)